src/contrib/sort/sort.cc

```diff
@@ -0,0 +1,65 @@
+/*!
+ *  Copyright (c) 2017 by Contributors
+ * \file Use standard C library call.
+ */
+
+#include "sort.h"
+#include <tvm/runtime/registry.h>
+#include <tvm/runtime/util.h>
+#include <dmlc/logging.h>
+#include <algorithm>
+#include <vector>
+
+namespace tvm {
+namespace contrib {
+
+using namespace runtime;
+
+template<>
+bool SortElem<float>::is_descend = true;
+
+// C library sort.
+// Sort a 3-D tensor for axis 1.
+// Number of element to be sorted for each sample can be specified
+// by num_elem tensor.
+// Index of value to be used for sorting is defined by sort_value_index.
+// Return index of sorted elements in each sample.
+TVM_REGISTER_GLOBAL("tvm.contrib.sort.stable_sort")
```

Comment:
If we follow the numpy convention and returns the index, it should be https://docs.scipy.org/doc/numpy/reference/generated/numpy.argsort.html